tensorflow/python/platform/test.py

```diff
@@ -95,6 +97,18 @@ def is_built_with_rocm():
   """Returns whether TensorFlow was built with ROCm (GPU) support."""
   return _test_util.IsBuiltWithROCm()
 
+@tf_export('test.disable_for_rocm')
```

Comment:
This will add tf.test.disable_for_rocm to the TF API. If that's what you're aiming for, you'll need to update the API goldens with these instructions: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/api/tests/README.txt

It'll also go through an API review.